app/controllers/api/member_authentications_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Api::MemberAuthenticationsController < ApplicationController
```

Comment:
I'm a bit worried about the `Api` namespace here since it's not really part of the API.